docs/section10/10-3.md

```diff
@@ -0,0 +1,40 @@
+# Airside systems – Rule 10-3
+
+**Rule ID:** 10-3
+ 
+**Rule Description:** For systems serving computer rooms, the baseline building design shall not have reheat for the purpose of dehumidification.  
+**Rule Assertion:** B-RMR = expected value                                           
+**Appendix G Section:** Section G3.1-10 HVAC Systems for the baseline building  
+**Appendix G Section Reference:** None
+**Data Lookup:** None  
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem  
+
+**Applicability Checks:** 
+
+1. The hvac system in B-RMR serves a computer room.
+
+**Function Call:** 
+
+1. get_zones_computer_rooms
+
+**Manual Check:** Check that the applicable baseline HVAC systems (hvac systems that serve comuter rooms) were modeled per the rules of G3.1-10 HVAC System baseline column which states that for systems serving computer rooms, the baseline building design shall not have reheat for the purpose of dehumidification.  
+ 
+
+**Rule Logic:**
+**Applicability Check 1:** 
+- Call function to get list of applicable zones in P_RMR: `applicable_zones_comp_room_p = get_zones_computer_rooms(P_RMR)`
+- For each zone in P_RMR: `zone_p in P_RMR...zones:`
+    - Check if the zone includes a computer room, if so then applicability flag is true: `if zone_p.id in applicable_zones_comp_room_p: rule_applicability_flag = TRUE`
+    - If rule_applicability_flag = TRUE for zone_p: `if rule_applicability_flag == TRUE:`
+        - Get analogous zone ID in the B_RMR: `zone_b = match_data_element(B_RMR,Zones,zone_p.id)`
+        - For each terminal unit associated with the zone_b in the B_RMR: `for terminal_b in zone_b:`
+            - Get the served_by_heating_ventilation_air_conditioning_systems for each terminal: `heating_ventilation_air_conditioning_systems_b = terminal_b.served_by_heating_ventilation_air_conditioning_systems`
+            - Add to list of heating_ventilation_air_conditioning_systems_list_b as the code loops through the terminal units: `heating_ventilation_air_conditioning_systems_list_b = heating_ventilation_air_conditioning_systems_list_b.append(heating_ventilation_air_conditioning_systems_b)`                
+- Convert the list of heating_ventilation_air_conditioning_systems_list_b associated with the applicable zones to a set and the back to a list to eliminate duplicates after looping through all zones: `heating_ventilation_air_conditioning_systems_list_b = list(set(heating_ventilation_air_conditioning_systems_list_b))`
+- For each hvac_b in heating_ventilation_air_conditioning_systems_list_b: `For in heating_ventilation_air_conditioning_systems_list_b:` 
+    **Rule Assertion:**
+    - Case 1: For each applicable hvac system, outcome is undetermined, manual check needs to be performed: `UNDETERMINED`
+    
```

Comment:
This looks good, we could potentially add an outcome message which provides a list of HVAC systems which could not be verified for rule ID 10-3. Something like '_HVAC System ID [X] serves a computer room and should not use reheat for dehumidification. This needs to be manually verified'_

Reply:
Addressed

Reply:
It seems that we could fairly easily verify it as suggested above. So I would consider making this rule mandatory.

Reply:
@mkarpman we need to discuss the solution you've proposed, would be good to make this rule mandatory